phpBB/config/default/container/services.yml

```diff
@@ -116,6 +116,12 @@ services:
             - @controller.resolver
             - @request_stack
 
+    language:
```

Comment:
I think that we should split it into at least two services:
- language_loader which will find the language files and load them when requested (basically it's a stateless service (minus caching) which as a `load(<component>, <locale>) : array('key' => 'value')` method or something similar).
- language which will take a loader as parameter. This service will be used by everyone.